tests/characters/control/blockchain/test_rpc_control_b.py

```diff
@@ -0,0 +1,107 @@
+import pytest
```

Comment:
Are these new intentionally test modules (added the letter `b`)?

Reply:
Somehow pytest isn't happy when they have the same filename, albeit in different directories. Thus, `_b` / `_f`

Reply:
Can we combine them into a single test - or does this need to be a stand alone module?